README.md

```diff
@@ -16,12 +16,12 @@ projects branches.
 As an example, the `containerized-tests` Jenkins job consists out of the
 following files:
 
-- `containerized-tests.yaml` is a [Jenkins Job Builder](jjb) configuration
+- `jobs/containerized-tests.yaml` is a [Jenkins Job Builder][jjb] configuration
   that describes the events when the job should get run and fetches the
   `.groovy` file from the git repository/branch
-- `containerized-tests.groovy` is the [Jenkins Pipeline](pipeline) that
+- `containerized-tests.groovy` is the [Jenkins Pipeline][pipeline] that
   contains the stages for the Jenkins Job itself. In order to work with [the
-  bare-metal machines from the CentOS CI](centos_ci), it executes the following
+  bare-metal machines from the CentOS CI][centos_ci], it executes the following
```

Comment:
the link is not generated here

Reply:
I guess the bracket pattern will be the same as the previous version @nixpanic

Reply:
`[centos_ci]` is not a valkid id, `[centos_ci_hw]` is listed at the bottom of the file